pkgs/shells/fish/default.nix

```
@@ -128,22 +127,22 @@ let
       sed -r "s|command grep|command ${gnugrep}/bin/grep|" \
           -i "$out/share/fish/functions/grep.fish"
       sed -i "s|which |${which}/bin/which |"               \
-              "$out/share/fish/functions/type.fish"
+             "$out/share/fish/functions/type.fish"
       sed -e "s|\|cut|\|${coreutils}/bin/cut|"             \
           -i "$out/share/fish/functions/fish_prompt.fish"
       sed -e "s|gettext |${gettext}/bin/gettext |"         \
           -e "s|which |${which}/bin/which |"               \
           -i "$out/share/fish/functions/_.fish"
       sed -e "s|uname|${coreutils}/bin/uname|"             \
           -i "$out/share/fish/functions/__fish_pwd.fish"   \
-            "$out/share/fish/functions/prompt_pwd.fish"
+             "$out/share/fish/functions/prompt_pwd.fish"
       sed -e "s|sed |${gnused}/bin/sed |"                  \
           -i "$out/share/fish/functions/alias.fish"        \
-            "$out/share/fish/functions/prompt_pwd.fish"
+             "$out/share/fish/functions/prompt_pwd.fish"
       sed -i "s|nroff |${groff}/bin/nroff |"               \
-            "$out/share/fish/functions/__fish_print_help.fish"
+             "$out/share/fish/functions/__fish_print_help.fish"
       sed -i "s|/sbin /usr/sbin||" \
```

As previously mentioned the line was modified in 3.1.0 to be

```fish
set -lx -a PATH /usr/local/sbin /sbin /usr/sbin
```

So this is going to leave it as

```fish
set -lx -a PATH /usr/local/sbin
```

To that end, this should be modified as

```suggestion
      sed -i "s|/usr/local/sbin /sbin /usr/sbin||" \
```

---

Would it be a better idea to just `sed -i "|/usr/local/sbin /sbin /usr/sbin|d"` (delete the matching line) then? No reason to keep it around if it doesn't do anything...

---

After some testing, I realized that `set -lx -a PATH` is a noop, so I'll just go with your suggestion for now.